e2e/interchaintestv8/relayer_test.go

```diff
@@ -0,0 +1,93 @@
+package main
+
+import (
+	"context"
+	"encoding/hex"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+
+	"github.com/ethereum/go-ethereum/crypto"
+
+	"github.com/srdtrk/solidity-ibc-eureka/e2e/v8/operator"
+	"github.com/srdtrk/solidity-ibc-eureka/e2e/v8/relayer"
+	"github.com/srdtrk/solidity-ibc-eureka/e2e/v8/testvalues"
+	relayertypes "github.com/srdtrk/solidity-ibc-eureka/e2e/v8/types/relayer"
+)
+
+// IbcEurekaTestSuite is a suite of tests that wraps TestSuite
+// and can provide additional functionality
+type RelayerTestSuite struct {
+	IbcEurekaTestSuite
```

Comment:
I want to keep them separate. We already use inheritance for IbcEurekaTestSuite.

I think inheritance is totally fine since these are just tests and it makes our lives simpler.

Reply:
Yeah, but 2 levels of inheritance is starting to get a bit crazy :) That said, maybe the fact that we would want to inherit from the IbcEurekaTestSuite just means that we should put all of that into the base level TestSuite. Why keep multiple levels if we always just need the same stuff. But we don't have to deal with that in this PR ofc

Reply:
Yeah, let's deal with it in another PR. I actually have another PR coming up which doesn't inherit from IbcEurekaTestSuite but inherits from TestSuite because it only deploys the sp1-ics07 client.

You should inherit `IbcEurekaTestSuite` if you need to deploy all the IbcEureka contracts. The relayer test suite needs this, so it inherits it. But other test suites might want to test other contracts/features in isolation